docs/community/tools/strands-deepgram.md

```diff
@@ -0,0 +1,53 @@
+# strands-deepgram
+
+{{ community_contribution_banner }}
+
+[strands-deepgram](https://pypi.org/project/strands-deepgram/) is a production-ready speech and audio processing tool for [Strands Agents SDK](https://github.com/strands-agents/sdk-python), powered by Deepgram's AI platform with 30+ language support.
```

Comment:
Can we remove the links to Strands-Agent SDK?  Given that we're on the docs, I don't think we need them.

Let's also have the first link be to the github project - I feel like most developers are going to be judging packages based on GitHub